instance.go

```diff
@@ -51,7 +51,7 @@ type Instance struct {
 
 // NewInstance returns an http.Handler that can handle a single request.
 func NewInstance(wasmbytes []byte, opts ...InstanceOption) *Instance {
-	var i = new(Instance)
+	i := new(Instance)
```

Comment:
These `var i =` to `i :=` changes were all automatically applied when I saved the Go file. Assuming this is more idiomatic Go. 🤷‍♂️